spec/services/court_application_link_validator_spec.rb

```diff
@@ -6,35 +6,21 @@
   let!(:court_application) do
     CourtApplication.create!(
       id: court_application_id,
+      subject_id:,
       body: JSON.parse(file_fixture("court_application_summary.json").read),
     )
   end
 
-  context "when there is a defendant offence" do
-    before do
-      CourtApplicationDefendantOffence.create!(court_application_id:,
-                                               defendant_id: subject_id,
-                                               offence_id: "cacbd4d4-9102-4687-98b4-d529be3d5710",
-                                               application_type: "appeal")
-    end
-
-    it "returns true" do
-      expect(link_validator_response).to be true
-    end
-
-    context "when the hearing summary does not exist" do
-      before do
-        court_application.body.delete("hearingSummary")
-        court_application.save!
-      end
+  it "returns true" do
+    expect(link_validator_response).to be true
+  end
```

Comment:
Nit:
`link_validator_response` is the subject, you could do:

`it { is_expected.to be true }`